### PR TITLE
Added logging configuration document

### PR DESCRIPTION
**Pull request by siy**

### CODE REVIEW

File: docs/logging.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+local or small scale deployments. Such configurations usually have logs stored in text log files and analyzed 
+manually with tools like `grep` or `awk`.
+#### Changing Default Logging Level
+Default configuration uses `INFO` logging level, which is moderately verbose. This logging level can be changed without
+updating configuration files by setting environment variable `RADIXDLT_LOG_LEVEL` to necessary logging level.    
+
+#### Logging with Dedicated Logging Infrastructure
+Complex deployments as well as deployments into cloud-based infrastructure usually rely on centralized 
+log collection and analysis tools like Splunk, AWS Centralized Logging or Loggly.
+For such use cases it's usually more convenient to have other format for logs that plain text. Most popular approach is
```
Comment (LukasGasior1) [inline docs/logging.md]: that -> than

**Review by LukasGasior1 — APPROVED** [review]
LGTM

File: docs/logging.md (hunk 1 above)
Comment (siy) [reply docs/logging.md]: Thanks. Fixed.

**Review by shambupujar — APPROVED** [review]
Looks good

**Pull request merged**